src/eventer/eventer_SSL_fd_opset.c

```diff
@@ -1115,18 +1118,18 @@ void eventer_ssl_init() {
   SSL_library_init();
   OpenSSL_add_all_ciphers();
 
-  if (!dh512_file || strcmp(dh512_file, "")) {
+  if (!dh1024_file || strcmp(dh1024_file, "")) {
     e = eventer_alloc();
     e->mask = EVENTER_ASYNCH;
     e->callback = generate_dh_params;
-    e->closure = (void *)512;
+    e->closure = (void *)1024;
     eventer_add_asynch(NULL, e);
   }
-  if (!dh1024_file || strcmp(dh1024_file, "")) {
+  if (!dh2048_file || strcmp(dh2048_file, "")) {
```

Comment:
We haven't had a way to disable them because they were so nominally cheap...

And it would appear that setting it to "" will disable the generation (AFAICT).  That's acceptable.